generate.go

```diff
@@ -1,4 +1,38 @@
-//go:generate go-bindata -pkg=data -ignore=.*\.go -o data.go . ./benkeen/d3pie/ ./jiahuang/d3-timeline/ ./krisk/fuse/ ./zeroclipboard/zeroclipboard/ ./seiyria/bootstrap-slider/
-//go:generate go fmt .
+// +build generate
 
-package data
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+	pathpkg "path"
+
+	"github.com/shurcooL/go/vfs/httpfs/filter"
+	"github.com/shurcooL/vfsgen"
+)
+
+func main() {
+	ignore := func(fi os.FileInfo, path string) bool {
+		switch {
+		case len(fi.Name()) >= 2 && fi.Name()[0] == '.': // Skip .files/.folders (but not ".").
+			return true
+		case pathpkg.Ext(fi.Name()) == ".go":
+			return true
+		case path == "/README.md":
+			return true
+		default:
+			return false
+		}
+	}
+	var data http.FileSystem = filter.NewIgnore(http.Dir("."), ignore)
+
+	err := vfsgen.Generate(data, vfsgen.Options{
+		Filename:     "data.go",
+		PackageName:  "data",
+		VariableName: "Assets",
+	})
+	if err != nil {
+		log.Fatalln(err)
+	}
+}
```

Comment:
With that change, here's how the folder layout and the `generate.go` file would look like:

![image](https://cloud.githubusercontent.com/assets/1924134/10000650/55431d9c-6052-11e5-8153-94f9deb8739a.png)